vocab_files/methods_by_module/camera-traps/collection.ttl

```diff
@@ -303,21 +293,16 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
   Figure. The detection of fauna by camera traps is affected by ecological and
   observational processes occurring at the broader, landscape scale and local,
   camera trap detection zone scale (Burton <em>et al.</em> 2015).
-  <table>
+  </p><table>
     <tbody>
       <tr>
         <td>
-          <img
-            src="https://w3id.org/tern/static/dawe-rlp/latest/camera-traps/detection-fauna-camera-traps.png"
-            alt="The detection of fauna by camera traps is affected by ecological and observational processes occurring"
-            widteh="620"
-            height="548"
-          />
+          <img src="https://w3id.org/tern/static/dawe-rlp/latest/camera-traps/detection-fauna-camera-traps.png" alt="The detection of fauna by camera traps is affected by ecological and observational processes occurring" widteh="620" height="548"/>
         </td>
       </tr>
     </tbody>
   </table>
-</p>
+<p/>
```

Comment:
🤔 is this valid?

Reply:
It's shown correctly on browser, but I'll change them back to </p>